<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="8" passed="5" failed="1" skipped="2">
  <reporter-output>
  </reporter-output>
  <suite started-at="2020-12-23T18:45:39Z" name="Suite" finished-at="2020-12-23T18:45:39Z" duration-ms="189">
    <groups>
      <group name="Sanity">
        <method signature="Testing3.deleteUser()[pri:3, instance:tests.Testing3@4009e306]" name="deleteUser" class="tests.Testing3"/>
      </group> <!-- Sanity -->
      <group name="Smoke">
        <method signature="Testing3.editUser()[pri:2, instance:tests.Testing3@4009e306]" name="editUser" class="tests.Testing3"/>
        <method signature="Testing3.deleteUser()[pri:3, instance:tests.Testing3@4009e306]" name="deleteUser" class="tests.Testing3"/>
        <method signature="Testing2.test4()[pri:0, instance:tests.Testing2@7dc0f706]" name="test4" class="tests.Testing2"/>
      </group> <!-- Smoke -->
    </groups>
    <test started-at="2020-12-23T18:45:39Z" name="Test" finished-at="2020-12-23T18:45:39Z" duration-ms="189">
      <class name="tests.Testing3">
        <test-method signature="createUser()[pri:1, instance:tests.Testing3@4009e306]" started-at="2020-12-23T13:45:39Z" name="createUser" finished-at="2020-12-23T13:45:39Z" duration-ms="47" status="FAIL">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Verifying element expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Verifying element expected [true] but found [false]
	at org.testng.Assert.fail(Assert.java:93)
	at org.testng.Assert.failNotEquals(Assert.java:512)
	at org.testng.Assert.assertTrue(Assert.java:41)
	at tests.Testing3.createUser(Testing3.java:15)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- createUser -->
        <test-method signature="editUser()[pri:2, instance:tests.Testing3@4009e306]" started-at="2020-12-23T13:45:39Z" name="editUser" finished-at="2020-12-23T13:45:39Z" depends-on-methods="tests.Testing3.createUser" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Testing3.editUser()[pri:2, instance:tests.Testing3@4009e306] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Testing3.editUser()[pri:2, instance:tests.Testing3@4009e306] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- editUser -->
        <test-method signature="deleteUser()[pri:3, instance:tests.Testing3@4009e306]" started-at="2020-12-23T13:45:39Z" name="deleteUser" finished-at="2020-12-23T13:45:39Z" depends-on-methods="tests.Testing3.editUser" duration-ms="0" status="SKIP">
          <exception class="java.lang.Throwable">
            <message>
              <![CDATA[Method Testing3.deleteUser()[pri:3, instance:tests.Testing3@4009e306] depends on not successfully finished methods]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.Throwable: Method Testing3.deleteUser()[pri:3, instance:tests.Testing3@4009e306] depends on not successfully finished methods
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1084)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.Throwable -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- deleteUser -->
      </class> <!-- tests.Testing3 -->
      <class name="tests.Testing1">
        <test-method is-config="true" signature="beforeSuite()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeSuite" finished-at="2020-12-23T13:45:39Z" duration-ms="13" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method is-config="true" signature="beforeTest()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeTest" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeTest -->
        <test-method is-config="true" signature="beforeClass()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeClass" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test1()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="test1" finished-at="2020-12-23T13:45:39Z" duration-ms="8" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test1 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test2()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="test2" finished-at="2020-12-23T13:45:39Z" duration-ms="3" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test2 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="beforeMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="beforeMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="1" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method signature="test3()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="test3" finished-at="2020-12-23T13:45:39Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test3 -->
        <test-method is-config="true" signature="afterMethod()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterMethod" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method is-config="true" signature="afterClass()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterClass" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterClass -->
        <test-method is-config="true" signature="afterTest()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterTest" finished-at="2020-12-23T13:45:39Z" duration-ms="0" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method is-config="true" signature="afterSuite()[pri:0, instance:tests.Testing1@3d34d211]" started-at="2020-12-23T13:45:39Z" name="afterSuite" finished-at="2020-12-23T13:45:39Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- tests.Testing1 -->
      <class name="tests.Testing2">
        <test-method signature="test4()[pri:0, instance:tests.Testing2@7dc0f706]" started-at="2020-12-23T13:45:39Z" name="test4" finished-at="2020-12-23T13:45:39Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test4 -->
        <test-method signature="test5()[pri:0, instance:tests.Testing2@7dc0f706]" started-at="2020-12-23T13:45:39Z" name="test5" finished-at="2020-12-23T13:45:39Z" duration-ms="2" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- test5 -->
      </class> <!-- tests.Testing2 -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
